prerender.js

```diff
@@ -1,29 +1,48 @@
-const fse = require('fs-extra')
-const path = require('path')
+import fse from 'fs-extra'
+import { dirname, resolve } from 'path'
+import { fileURLToPath } from 'url'
+import { createServer as createViteServer } from 'vite'
+
+const dir = dirname(fileURLToPath(import.meta.url))
```

Comment:
These changes are just to replace `__dirname`, see https://gist.github.com/sindresorhus/a39789f98801d908bbc7ff3ecc99d99c#what-do-i-use-instead-of-__dirname-and-__filename